README.md

```diff
@@ -1 +1,34 @@
-# AI_Conference-CapstoneProject
\ No newline at end of file
+# AI_Conference-CapstoneProject
+  This Web page is a the first Capstone project for Microverse first module. Itis about the importance of Artificial Intelligence in our life. It designed according to Behance Cindy Shine Design.
+
+# Build With 👩‍🔧
+.HTML
+
+.CSS
+
+.JavaScript
+
+.Sublime
+
+# Getting Started 🎬
+In this project I create a Headline page, Cards, About me page. It has a menu bar which is expandable, a popup menu for both mobile and desktop virsion and button interaction.
+
+# Author
+Sahar Saba Amiri
+
+. Gmail: saharsaba.amiri123@gmail.com
+
+. LinkedIn: https://www.linkedin.com/sahar-saba-amiri
+
+. GitHub : https://github.com/Sahar-SE
+
+# Contributing
+Contributions, issues, and feature requests are welcome!
+
+# links
+The link of my Portfolio GitHub io is: 
+## Pls remember that github pages has issues frequently, this does not reflect on my code or work!
+ https://sahar-se.github.io/AI_Conference-CapstoneProject/
+
+# Show your Support
+By giving me a Star✨
```

Comment:
- [ ] [OPTIONAL] Although not pointed out by the previous reviewer, but I strongly suggest you include an acknowledgement for Cindy Shin the owner of the original design.